src/graph/GraphViewServer.ts

```diff
@@ -11,12 +11,45 @@ import * as io from 'socket.io';
 import { setInterval } from 'timers';
 import { GraphConfiguration } from './GraphConfiguration';
 import * as gremlin from "gremlin";
+import { removeDuplicatesById } from "../utils/array";
+
+let maxVertices = 300;
+let maxEdges = 1000;
+
+interface Edge {
+  id: string;
+  type: "edge";
+  inV: string;  // Edge source ID
+  outV: string; // Edge target ID
+};
+
+interface Vertex {
+  id: string;
+  type: "edge";
```

Comment:
Is a vertex really supposed to be type edge?

Reply:
Thanks.